### PR TITLE
Improving help centre

### PR DESCRIPTION
**Pull request by tristanhcole**
**Describe the Pull Request**
A few small fixes and improvements to our help centre, namely:
- renames IntercomHelpCentre to just HelpCentre
- changes href to point to new docs.withsempo.com
- passing subdomain to intercom so that we can use things like Product Tours in app

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

**Comment by tristanhcole** [comment]
[Update help centre on app to point to docs.withsempo.com](https://trello.com/c/E3xswkfv/595-update-help-centre-on-app-to-point-to-docswithsempocom)

**Comment by tristanhcole** [comment]
[Pass app subdomain to intercom](https://trello.com/c/Gab2ClX7/593-pass-app-subdomain-to-intercom)

File: app/client/components/intercom/HelpCentre.tsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import * as React from "react";
 import { LinkOutlined } from "@ant-design/icons";
 
-export const IntercomHelpCentre = () => {
+export const HelpCentre = () => {
   return (
-    <a href="https://help.withsempo.com/" target="_blank">
+    <a href="http://docs.withsempo.com/" target="_blank">
```
Comment (enjeyw) [inline app/client/components/intercom/HelpCentre.tsx:6]: http?

**Review by michielderoos — APPROVED** [review]
Looks good, once link is HTTPS'd

**Pull request merged**